### ISSUE TITLE
강의 노트 번역 파일들

### ISSUE THREAD

**Issue by myungsub** [aikorea/cs231n#9]
### Module 1: Neural Networks

각 강의 노트마다 번역해주셔야 할 파일은 다음과 같습니다.
- Image Classification: Data-driven Approach, k-Nearest Neighbor, train/val/test splits
  - classification.md
- Linear classification: Support Vector Machine, Softmax
  - linear-classify.md
- Optimization: Stochastic Gradient Descent
  - optimization-1.md
- Backpropagation, Intuitions
  - optimization-2.md
- Neural Networks Part 1: Setting up the Architecture
  - neural-networks-1.md
- Neural Networks Part 2: Setting up the Data and the Loss
  - neural-networks-2.md
- Neural Networks Part 3: Learning and Evaluation
  - neural-networks-3.md

**Comment by myungsub** [comment]
optimization-1.md : 임준구
optimization-2.md : 박기루
이렇게 맡아주시기로 했습니다.

**Comment by stats2ml** [comment]
neural-networks-1.md : 임준구
추가로 하나 더 해보겠습니다.

**Comment by younggeunchoi** [comment]
neural-networks-3.md : 최영근 (slack : ygchoi) 하겠습니다:)

**Comment by salopge** [comment]
neural-networks-2.md
아직 맡으신 분 없으면 제가 하겠습니다~

**Comment by jazzsaxmafia** [comment]
convolutional-networks.md는 제가 하겠습니다.
일단 fork 한 뒤 https://github.com/jazzsaxmafia/cs231n/blob/master/convolutional-networks-korean.md 에 정리하고 있고요, 완성 되면 PR 할게요
